setup.py

```diff
@@ -5,7 +5,7 @@
 install_requires = (local_path / "requirements.txt").read_text().splitlines()
 
 setup(name="gretel-trainer",
-      version="0.2.1",
+      version="0.2.2",
```

Comment:
we should change this to pull from github tags instead of manually doing it - we can set that up in a separate pr

Reply:
that's good, never liked doing that in code 👍